Thunder/utils/canonical_files.py

```
@@ -0,0 +1,463 @@
+                    _remember(record)
+                    return record, stored_message, False
+                except DuplicateKeyError:
+                    reusable_record = await _wait_for_other_worker_canonical_record(file_unique_id)
+                    if reusable_record:
+                        schedule_touch_file_record(reusable_record, reused=True)
+                        return reusable_record, stored_message, True
+                    if stored_message:
+                        try:
+                            await stored_message.delete()
+                        except Exception:
+                            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)